2019.12/apache/Dockerfile

```diff
@@ -11,9 +11,19 @@ RUN set -ex; \
         git \
 # For mail() support
         ssmtp \
+# For tini installation
+        gnupg dirmngr \
     ; \
     rm -rf /var/lib/apt/lists/*;
 
+# Add tini for reaping processes
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-${ARCH:-amd64} /sbin/tini
```

Comment:
Use `$debMultiarch` for architecture type
See:
https://github.com/docker-library/php/blob/a80762e229981b94c9fe6c381637350e9104096a/Dockerfile-debian.template#L139

Reply:
I had a look to this one but was returning `x86_64-linux-gnu` instead of `amd64`
it looks like the good one is `DEB_BUILD_ARCH`

Reply:
ah ok .. sorry you're right

Reply:
done :)